nixos/modules/virtualisation/docker.nix

```diff
@@ -229,7 +230,7 @@ in
       };
 
       assertions = [
-        { assertion = cfg.enableNvidia && pkgs.stdenv.isx86_64 -> config.hardware.opengl.driSupport32Bit or false;
+        { assertion = cfg.enableNvidia && pkgs.stdenv.isx86_64 && !cfg.package.supports-cdi -> config.hardware.opengl.driSupport32Bit or false;
```

Comment:
`or false`: is it ever not declared?

Reply:
won't notice the error if one removes the option later?

Reply:
I just added the CDI check to keep the same behavior as before. I'm not sure if driSupport32Bit is required for CDI, I don't think so.

Reply:
Didn't notice it was already there; not our business then, let's keep as is